_posts/2025-03-10-data-wants-to-be-free-japanese.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+lang: ja-JP
+title: "データは自由になりたい：Apache Arrowでデータ交換は高速"
+description: ""
+date: "2025-02-28 00:00:00"
+author: David Li, Ian Cook, Matt Topol
+categories: [application]
+image:
+  path: /img/arrow-result-transfer/part-1-share-image.png
+  height: 1200
+  width: 705
+translations:
+  - language: 原文（English）
+    post_id: 2025-02-28-data-wants-to-be-free
+---
+
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+<style>
+.a-header {
+  color: #984EA3;
+  font-weight: bold;
+}
+.a-data {
+  color: #377EB8;
+  font-weight: bold;
+}
+.a-length {
+  color: #FF7F00;
+  font-weight: bold;
+}
+.a-padding {
+  color: #E41A1C;
+  font-weight: bold;
+}
+</style>
+
+_この記事はデータベースとクエリーエンジン間のデータ交換フォーマットとしてなぜArrowが使われているのかという謎を解くシリーズの２記事目です。_
+
+{% include arrow_result_transfer_series_japanese.md %}
+
+データ技術者として、データはよく「人質に取られた」ことをわかっています。
+データをもらい次第データを使わずに、まず時間を掛けなければいけません。
```

Comment:
How about splitting to two simple sentences instead of one sentence?

```suggestion
データをもらってもすぐに使うことはできません。使えるようになるまでに時間がかかります。
```

In general, we don't use Kanji for "掛ける" in "時間を掛ける". We use Hiragana for it.